Participate/Asia-Pacific-community-call.md

```diff
@@ -3,4 +3,4 @@
 
 This working group aims to connect with community members in the Asia Pacific region regarding open source community health. Discussions will focus on recaping CHAOSS work as well as identifying new areas of interest. The working group is, of course, open to everyone from any global region. 
 
-The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/Chicago/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call#eyJ1cmwiOiJodHRwczovL3Vub21haGEuem9vbS51cy9qLzcyMDQzMTI4OCJ9)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
+The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/(GMT-5:00)%20Central%20Time%20(US%20and%20Canada)/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```

Comment:
It looks like there was a regression in the last commit.

This should fix it again:

```suggestion
The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/Chicago/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```